docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+version: '3'
+
+services:
+    php:
+        depends_on:
+          - elasticsearch1
+          - pgsql
+          - mariadb-server
+        image: rollerworks/search-dev:latest
+        volumes:
+            - .:/srv/www:rw
+
+    # Storage services
+    pgsql:
+        image: postgres:9.6-alpine
+        environment:
+            - POSTGRES_DB=search_tests
+            - POSTGRES_USER=postgres
+        volumes:
+            - pg-data:/var/lib/postgresql/data:rw
+        ports:
+            - "5432:55432"
+
+    mariadb-server:
+        image: mariadb
+        restart: always
+        environment:
+            - MYSQL_DATABASE=search_tests
+            - MYSQL_ROOT_PASSWORD=nope
+        volumes:
+            - my-data:/var/lib/mysql:rw
+        ports:
+            - "3306:53306"
+
+    elasticsearch1:
+        image: docker.elastic.co/elasticsearch/elasticsearch:5.6.3
+        environment:
+            - "discovery.type=single-node"
+            - "xpack.security.enabled=false"
+            - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+        volumes:
+            - esdata:/usr/share/elasticsearch/data
+        ports:
+            - "9200:9200"
```

Comment:
Note, this was not prefixed with 5. :(

Reply:
For some reasons this broke the tests 😐

Reply:
I know, tried to change it. :cry: Will submit a PR to fix it once I figure out what it is.

Reply:
It's over 9200 😂 https://stackoverflow.com/questions/113224/what-is-the-largest-tcp-ip-network-port-number-allowable-for-ipv4

The port number seems to high.

Reply:
But... 59200 < 65535? That's why I proposed 5 as a prefix, and not 6, for example.

Reply:
> Either specify both ports (HOST:CONTAINER), or just the container port (a random host port will be chosen).

See #200